app/build.gradle

```diff
@@ -44,6 +44,9 @@ dependencies {
     // Timber
     implementation 'com.jakewharton.timber:timber:5.0.1'
 
+    // viewBinding delegate
+    implementation 'com.github.kirich1409:viewbindingpropertydelegate-noreflection:1.5.0-beta01'
```

Comment:
maybe use not a beta version? (I'm not sure how good is beta) 

But if it is okay - leave it